addons/binding/org.openhab.binding.hdpowerview/src/main/java/org/openhab/binding/hdpowerview/internal/api/requests/ShadeMove.java

```diff
@@ -19,7 +19,7 @@
 
     ShadeIdPosition shade;
 
-    public ShadeMove(String id, ShadePosition position) {
-        this.shade = new ShadeIdPosition(id, position);
+    public ShadeMove(int intid, ShadePosition position) {
```

Comment:
Can the name just remain `id` otherwise it should be `intId` according to the coding guidelines of Java

Reply:
I would just remove the int from it. It's easy to determine the type with a strong typed language like Java. :-)